ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -494,3 +495,28 @@ def test_ipa_certs_check_ipacertnsstrust(self):
                 check["kw"]["key"] in DEFAULT_PKI_KRA_CERTS
             )
         tasks.uninstall_master(self.master)
+
+
+class TestIpaHealthCheckOnReplica(IntegrationTest):
+
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master, setup_dns=True)
+        tasks.install_replica(cls.master, cls.replicas[0], setup_dns=True)
+
+    def test_run_with_stopped_master(self):
+        """
+        Test output of healthcheck where master IPA services are stopped
+        contains only errors regarding master being stopped and no other false
+        positives.
+        """
+        self.master.run_command(["ipactl", "stop"])
```

Comment:
This will leave IPA stopped once the test is done. Can you add a fixture to stop then restart IPA?